packages/terra-breakpoints/docs/README.md

```diff
@@ -0,0 +1,149 @@
+# Terra Breakpoints
+
+`terra-breakpoints` contains components and utilities related to Terra's supported responsive breakpoints.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-breakpoints`
+
+## Usage
+
+### Breakpoints 
+
+An object containing the named breakpoint values is the default export of the `terra-breakpoints` package. These named breakpoints are defined as minimum values.
+
+|Breakpoint Name|Minimum Value|
```

Comment:
I feel like we should expand this table to show the media query and description.

```
|Breakpoint Name|Minimum Value|Media Query|Description|
|---|---|---|---|
|`tiny`|0px|`@media screen and (min-width: 0px)`| Active from viewport width 0px and up |
|`small`|544px|`@media screen and (min-width: 544px)`|Active from viewport width 544px and up|
|`medium`|768px|`@media screen and (min-width: 768px)`|Active from viewport width 768px and up|
|`large`|992px|`@media screen and (min-width: 992px)`|Active from viewport width 992px and up|
|`huge`|1216px|`@media screen and (min-width: 1216px)`|Active from viewport width 1216 and up|
|`enormous`|1440px|`@media screen and (min-width: 1440px)`|Active from viewport width 1440px and up|
```